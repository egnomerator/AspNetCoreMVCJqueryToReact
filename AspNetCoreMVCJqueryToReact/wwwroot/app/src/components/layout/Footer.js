import React from "react";

export default class FooterMsgPart extends React.Component {
    render() {
        return '2021 - AspNetCoreMVCJqueryToReact'
    }
}