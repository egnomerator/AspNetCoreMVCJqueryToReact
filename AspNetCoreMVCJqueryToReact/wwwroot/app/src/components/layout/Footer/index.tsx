import * as React from "react";

export default class FooterMsgPart extends React.Component {
    render() {
        return <span>2021 - AspNetCoreMVCJqueryToReact</span>;
    }
}