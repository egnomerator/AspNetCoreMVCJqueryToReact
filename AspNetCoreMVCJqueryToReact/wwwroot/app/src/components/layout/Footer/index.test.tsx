import * as React from "react";
import * as ReactTestRenderer from "react-test-renderer";
import Footer from ".";

describe("Footer", () => {
    it("should render", () => {
        const footer = ReactTestRenderer.create(<Footer/>);

        expect(footer).toMatchSnapshot();
    });
});
