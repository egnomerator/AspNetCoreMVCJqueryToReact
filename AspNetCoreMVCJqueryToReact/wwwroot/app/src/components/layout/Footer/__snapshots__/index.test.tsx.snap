// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render 1`] = `
<span>
  2021 - AspNetCoreMVCJqueryToReact
</span>
`;
